fastlane_bot/events/pools/carbon_v1.py

```diff
@@ -79,7 +79,7 @@ def parse_event(
         """
         order0, order1 = CarbonV1Pool.parse_orders(event_args, event_type)
         data["cid"] = event_args["args"].get("id")
-        if isinstance(order0, list) and isinstance(order1, list):
+        if isinstance(order0, (list, tuple)) and isinstance(order1, (list, tuple)):
```

Comment:
Get rid of this type checking, and rewrite function `parse_orders` to always return the same type.
And while you're at it, since that function isn't used anywhere else, just implement it inside this function.